rollup.config.js

```diff
@@ -70,18 +70,57 @@ function createCommonJSConfig(input, output) {
   }
 }
 
+function createUMDConfig(input, output) {
+  return {
+    input,
+    output: {
+      file: output,
+      format: 'umd',
+      exports: 'named',
+      name: 'zustand',
+      esModule: true,
```

Comment:
https://rollupjs.org/guide/en/#outputesmodule
Is it true by default?

Reply:
Yeap, I'll remove it